libraries/wutdevoptab/devoptab_fsa_utimes.cpp

```diff
@@ -0,0 +1,9 @@
+#include "devoptab_fsa.h"
+
+int
+__wut_fsa_utimes(struct _reent *r,
```

Comment:
Same thing here - if the devoptab method only returns ENOSYS, it can be removed.